components/feature/readerview/src/main/assets/extensions/readerview/readability/readability-0.2.0.js

```diff
@@ -0,0 +1,1800 @@
+/*eslint-env es6:false*/
```

Comment:
This file could be minified too

Reply:
https://github.com/mozilla-mobile/android-components/issues/2288